src/hooks/useAutoFocusInput.ts

```diff
@@ -35,23 +34,20 @@ export default function useAutoFocusInput(): UseAutoFocusInput {
 
     useFocusEffect(
         useCallback(() => {
-            focusTimeoutRef.current = setTimeout(() => {
+            const timeoutRef = setTimeout(() => {
```

Comment:
Can you help explain what problem is solved by declaring the timeout inside the `useCallback` rather than using `useRef`?

Reply:
It's not related to performance itself but mostly to code simplicity. If timeout id does not need to be accessed by outside context then we could make it easier to reason about by just declaring const variable instead of assigning it to useRef instance. What do you think?